tools/spelling_dictionary.txt

```diff
@@ -310,6 +310,7 @@ WS
 Welford's
 Werror
 XDS
+ar
```

Comment:
can we change foo$b#$ar to foo$b#$are for all the various explantions of that transform to avoid this addition?  Feels like it'd be a common typo of "are" which we should catch.